change/@fluentui-react-05a7d92d-213d-45ca-b56d-e0d581556651.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updating Combobox snapshot",
+  "packageName": "@fluentui/react",
+  "email": "esteban.230@hotmail.com",
+  "dependentChangeType": "patch"
```

This should be none since it doesn't affect the published package.
```suggestion
  "type": "none",
  "comment": "Updating Combobox snapshot",
  "packageName": "@fluentui/react",
  "email": "esteban.230@hotmail.com",
  "dependentChangeType": "none"
```